src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -372,6 +390,7 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 ## Libraries
 
 - [web-bluetooth-utils](https://www.npmjs.com/package/web-bluetooth-utils) is a npm module that adds some convenience functions to the API.
+- A Web Bluetooth API shim is available in [noble](https://github.com/sandeepmistry/noble), the most popular Node.js BLE central module. This allows you to webpack/browserify noble without the need for a WebSocket server or other plugins.
```

Comment:
I don't understand what "allows you to webpack/browserify noble without ..." means. What's fixing what? I would have guessed this means that devs can use web bluetooth style code in non-browsers. Basically, what I can do with this that I couldn't before?